document/src/test/java/com/yahoo/document/update/TensorRemoveUpdateTest.java

```diff
@@ -10,17 +10,32 @@
 public class TensorRemoveUpdateTest {
 
     @Test
-    public void apply_remove_update_operations() {
-        assertApplyTo("{{x:0,y:0}:2, {x:0,y:1}:3}", "{{x:0,y:1}:1}", "{{x:0,y:0}:2}");
-        assertApplyTo("{{x:0,y:0}:1, {x:0,y:1}:2}", "{{x:0,y:0}:1,{x:0,y:1}:1}", "{}");
-        assertApplyTo("{}", "{{x:0,y:0}:1}", "{}");
-        assertApplyTo("{{x:0,y:0}:2, {x:0,y:1}:3}", "{}", "{{x:0,y:0}:2, {x:0,y:1}:3}");
+    public void apply_remove_update_operations_sparse() {
```

Comment:
Now that remove() is implemented in the tensor library I think we should move all the detailed tests there. Only the simple integration in applyTo() should be tested here.